### PR TITLE
[Merged by Bors] - feat(Topology/Algebra): add `TopologicalGroup.isInducing_iff_nhds_one`

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by alreadydone — APPROVED** [review]
maintainer merge

File: Mathlib/Topology/Algebra/Group/Basic.lean [hunk 1]
```diff
@@ -811,6 +811,17 @@ theorem continuous_of_continuousAt_one₂ {H M : Type*} [CommMonoid M] [Topologi
     (((hl x).comp tendsto_snd).mul hf)).mono_right (le_of_eq ?_)
   simp only [map_one, mul_one, MonoidHom.one_apply]
 
+@[to_additive]
+lemma TopologicalGroup.isInducing_of_nhds_one
+    {H : Type*} [Group H] [TopologicalSpace H] [TopologicalGroup H] (f : G →* H)
+    (hf : 𝓝 (1 : G) = (𝓝 (1 : H)).comap f) : Topology.IsInducing f := by
```
Comment (ocfnash) [inline Mathlib/Topology/Algebra/Group/Basic.lean:817]: How about we make this an iff?
```suggestion
@[to_additive]
lemma TopologicalGroup.isInducing_iff_nhds_one
    {H : Type*} [Group H] [TopologicalSpace H] [TopologicalGroup H] (f : G →* H) :
    Topology.IsInducing f ↔ 𝓝 (1 : G) = (𝓝 (1 : H)).comap f := by
```

**Review by ocfnash — COMMENTED** [review]
I think worth upgrading to an iff, otherwise LGTM.

bors d+

**Comment by erdOne** [comment]
bors merge

**Pull request closed without merging**